docs/CONTRIBUTING.rst

```diff
@@ -16,11 +16,6 @@ Types of Contributions
 Did you find a bug? or Do you intend to add a new feature or change an existing one?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-* **Identify the appropriate repository** for the change you are suggesting:
```

Comment:
I thin the "**Identify the appropriate repository**" section should be kept. The part we should clarify here, however, is that changes to the schema should be implemented and tested in PyNWB first. The changes can then be migrated to the nwb-schema repo once formally accepted. However, issue tickets regarding the schema should still be filed on the nwb-schema repo.